packages/eslint-plugin/tests/rules/no-unused-expressions.test.ts

```diff
@@ -325,5 +325,49 @@ Foo<string>;
         },
       ]),
     },
+    {
+      code: 'declare const foo: number | undefined; foo;',
+      errors: error([
+        {
+          line: 1,
+          endLine: 1,
+          column: 40,
+          endColumn: 44,
+        },
+      ]),
+    },
+    {
+      code: 'declare const foo: number | undefined; foo as any;',
+      errors: [
+        {
+          line: 1,
+          endLine: 1,
+          column: 40,
+          endColumn: 51,
+        },
+      ],
```

Comment:
```suggestion
      errors: error([
        {
          line: 1,
          endLine: 1,
          column: 40,
          endColumn: 51,
        },
      ]),
```

[Testing] Other `invalid` test cases use this `error` helper function. Let's use it for all newly added cases for better consistency?

Reply:
Hi! Thanks for the feedack! I'm going to take a look :)

Additionally, I just realised that my pipeline is failing on `lint with build`? Any command to run it just for the classes w/changes?